virtual_main.c

```diff
@@ -1643,6 +1643,10 @@ voss_rx_backend_refresh(void)
 #ifdef HAVE_BLUETOOTH
 	} else if (strstr(voss_dsp_rx_device, "/dev/bluetooth/") == voss_dsp_rx_device) {
 		voss_rx_backend = &voss_backend_bt_rec;
+#endif
+#ifdef HAVE_SNDIO
+	} else if (strstr(voss_dsp_tx_device, "sndio:") == voss_dsp_tx_device) {
```

Comment:
I wonder if we should use /dev/sndio/xxx for this? What do you think?

Reply:
Sounds like a good idea. I've made this change.